packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -105,13 +106,46 @@ class C4DMastheadComposite extends HostListenerMixin(LitElement) {
    */
   protected _renderL1() {
     const { l1Data, selectedMenuItemL1 } = this;
+
+    /**
+     * Creates CTA markup based on l1Data and slots it into the L1.
+     *
+     * While it would be better to encapsulate this behavior in the L1 itself, we
+     * are forced to render CTAs into the light DOM so that the IBM Contact Module
+     * app can find them.
+     *
+     * @returns A template fragment representing an L1 CTA or an empty string.
+     */
+    const renderL1Cta = (): TemplateResult | string => {
```

Comment:
What if this was be co-located with the L1 component by making it a static method there, passing the relevant `cta` data? Would it feel a bit more at home over there?

Reply:
That's so smart! Yes, that would make this feel 100x better to me. Going to do that right now.